session.go

```diff
@@ -37,20 +39,27 @@ func (s *Session) WithJwtKey(jwt string) *Session {
 	return s
 }
 
+// WithOrganization adds the organization to the session.
+func (s *Session) WithOrganization(key, secret string) *Session {
+	s.AppKey = key
+	s.AppSecret = []byte(secret)
+	return s
+}
+
 // WithClient sets a new client that will be used to make requests to the Hyper Solutions API.
 func (s *Session) WithClient(client *http.Client) *Session {
 	s.Client = client
 	return s
 }
 
-func (s *Session) generateSignature() (string, error) {
+func (s *Session) generateSignature(key string, jwtKey []byte) (string, error) {
```

Comment:
Small nit: This no longer needs to be a method since it does not access `Session`